src/RenderEngineManager.cc

```diff
@@ -75,15 +82,18 @@ class ignition::rendering::RenderEngineManagerPrivate
   /// \brief Engines that have been registered
   public: EngineMap engines;
 
-  /// \brief A map of default engine name to its plugin library name
+  /// \brief A map of default engine library names to their plugin names.
   public: std::map<std::string, std::string> defaultEngines;
 
-  /// \brief A map of loaded engine plugins to its plugin name
+  /// \brief A map of loaded engine plugins to its plugin name.
   public: std::map<std::string, std::string> enginePlugins;
 
-  /// \brief Plugin loader for managing render engine plugin libraries
+  /// \brief Plugin loader for managing render engine plugin libraries.
   public: ignition::plugin::Loader pluginLoader;
 
+  /// \brief Environment variable which holds paths to look for engine plugins.
+  public: std::string pluginPathEnv = "IGN_GAZEBO_RENDER_ENGINE_PATH";
```

Comment:
I think the `IGN_GAZEBO_RENDER_ENGINE_PATH` should not be harded in ign-rendering as that's an ign-gazebo thing. Maybe an API that lets users set the plugin search path? So in ign-gazebo, it reads the `IGN_GAZEBO_RENDER_ENGINE_PATH` env var and calls an ign-rendering to pass this path to the render engine manager